yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -784,46 +784,49 @@ func getAssessmentReportContentFromAnalyzeSchema() error {
 	return nil
 }
 
-func addUnsupportedFeaturesFromSchemaAnalysisReport(featureName string, issueReasons []string, schemaAnalysisReport utils.SchemaReport, unsupportedFeatures *[]UnsupportedFeature) {
+func addUnsupportedFeaturesFromSchemaAnalysisReport(featureName string, issueReason string, schemaAnalysisReport utils.SchemaReport, unsupportedFeatures *[]UnsupportedFeature, displayDDLInHTML bool, description string) {
```

Comment:
just to confirm, we reverted this change back to a single `issueReason` because we are no longer grouping multiple issues under "unsupported DDLs", is that correct?

Reply:
Right